tests/resources/session-tests-template/suite.xml

```diff
@@ -0,0 +1,233 @@
+<?xml version='1.0' encoding='UTF-8'?>
+<suite version="36" descriptor="Suite File">
+  <xform>
+    <resource id="1662072614e4dc0c3808d095c5b1cc826be7c1ec" version="28" descriptor="Form: (Module Mother) - Registration">
+      <location authority="local">./modules-0/forms-0.xml</location>
+    </resource>
+  </xform>
+  <xform>
+    <resource id="29f9e33b1cd347250d970ff5f9e5c91ad73a5337" version="28" descriptor="Form: (Module Mother) - Follow-Up 1">
+      <location authority="local">./modules-0/forms-1.xml</location>
+    </resource>
+  </xform>
+  <xform>
+    <resource id="49d0d92bbd0ed93f577cbfa6e3a38ec44cff59ba" version="28" descriptor="Form: (Module Mother) - Follow-Up 2">
+      <location authority="local">./modules-0/forms-2.xml</location>
+    </resource>
+  </xform>
+  <xform>
+    <resource id="26cf1e4fe9d3fc64a5df104e98db48ca9a469618" version="28" descriptor="Form: (Module Child) - Registration">
+      <location authority="local">./modules-1/forms-0.xml</location>
+    </resource>
+  </xform>
+  <xform>
+    <resource id="528470d90c2d87f6dc7ce52922df0dff495df645" version="36" descriptor="Form: (Module Child) - 1-Month Check-Up">
+      <location authority="local">./modules-1/forms-1.xml</location>
+    </resource>
+  </xform>
+  <xform>
+    <resource id="79440efa14a0fe3989fe712c7c05fe8e3f9ed921" version="45" descriptor="Form: (Module Untitled Module) - Untitled Form">
+      <location authority="local">./modules-2/forms-0.xml</location>
+    </resource>
+  </xform>
+  <xform>
+    <resource id="b5fb3b3a985700f8979ef3d175e183deadb23b5c" version="45" descriptor="Form: (Module Untitled Module) - Untitled Form 2">
+      <location authority="local">./modules-2/forms-1.xml</location>
+    </resource>
+  </xform>
+  <detail id="m0_case_short">
+    <title>
+      <text>
+        <locale id="m0.case_short.title"/>
+      </text>
+    </title>
+    <field>
+      <header>
+        <text>
+          <locale id="m0.case_short.case_name_1.header"/>
+        </text>
+      </header>
+      <template>
+        <text>
+          <xpath function="case_name"/>
+        </text>
+      </template>
+      <sort type="string" order="1" direction="ascending">
+        <text>
+          <xpath function="case_name"/>
+        </text>
+      </sort>
+    </field>
+  </detail>
+  <detail id="m0_case_long">
+    <title>
+      <text>
+        <locale id="m0.case_long.title"/>
+      </text>
+    </title>
+    <field>
+      <header>
+        <text>
+          <locale id="m0.case_long.case_name_1.header"/>
+        </text>
+      </header>
+      <template>
+        <text>
+          <xpath function="case_name"/>
+        </text>
+      </template>
+    </field>
+  </detail>
+  <detail id="m1_case_short">
+    <title>
+      <text>
+        <locale id="m1.case_short.title"/>
```

Comment:
These details are either unused or can be cleaned up, right, since these locale strings aren't being loaded? Would be good to remove as much of the extraneous stuff in here as possible to make it easy to extend/adapt.